diff -upNr glibmm-2.18.0.orign/gio/giomm/error.h glibmm-2.18.0/gio/giomm/error.h
--- glibmm-2.18.0.orign/gio/giomm/error.h	2008-07-25 21:11:43.000000000 +0200
+++ glibmm-2.18.0/gio/giomm/error.h	2008-10-05 13:10:19.000000000 +0200
@@ -27,6 +27,15 @@
 
 #include <glibmm/interface.h>
 
+// There have been issues with other libraries defining HOST_NOT_FOUND (e.g.
+// netdb.h).  As a workaround, we added the alternate name HOST_WAS_NOT_FOUND.
+// Portable code should not use HOST_NOT_FOUND.  Undefining it here (and
+// restoring it below) will allow programs to compile even if they include
+// netdb.h.  See Bug #529496
+#ifdef HOST_NOT_FOUND
+#define GIOMM_SAVED_HOST_NOT_FOUND HOST_NOT_FOUND
+#undef HOST_NOT_FOUND
+#endif // HOST_NOT_FOUND
 
 namespace Gio
 {
@@ -96,6 +105,11 @@ private:
 
 } // namespace Gio
 
+#ifdef GIOMM_SAVED_HOST_NOT_FOUND
+// restore the previously-defined HOST_NOT_FOUND macro
+#define HOST_NOT_FOUND GIOMM_SAVED_HOST_NOT_FOUND
+#undef GIOMM_SAVED_HOST_NOT_FOUND
+#endif // GIOMM_SAVED_HOST_NOT_FOUND
 
 #endif /* _GIOMM_ERROR_H */
 
